datumaro/components/dataset.py

```diff
@@ -732,14 +751,28 @@ def update(self,
         If the source is a dataset, labels are matched. If the labels match,
         but the order is different, the annotation labels will be remapped to
         the current dataset label order during updating.
+
+        Returns: self
         """
+
         self._data.update(source)
         return self
 
     def transform(self, method: Union[str, Type[Transform]],
-            *args, **kwargs) -> Dataset:
+            **kwargs) -> Dataset:
         """
         Applies some function to dataset items.
+
+        Results are stored in-place. Modifications are applied lazily.
+
+        Args:
+            method (str or Transform) - The transformation function to be
+                applied to the dataset. If a string is passed, it is treated
+                as a plugin name, which is searched for in
+                the dataset environment
+            **kwargs - Optional parameters for the transformation
```

Comment:
```suggestion
            **kwargs - Parameters for the transformation
```

I assume that transformations can have both optional and mandatory parameters, which will both be specified using this argument.

Reply:
Fixed such cases

Reply:
I think you missed `validate`.